client/state/signup/steps/website-content/schema.ts

```diff
@@ -41,22 +51,42 @@ export interface WebsiteContentCollection {
 }
 
 export const initialState: WebsiteContentCollection = {
+	currentIndex: 0,
+	websiteContent: [],
+};
+
+export const initialTestState: WebsiteContentCollection = {
```

Comment:
Felt like it's nice to have a example of the entire data structure inside the schema files. But will move it if you think it belongs more in the test files 👌

Reply:
I had to move the test state to the back here to the schema file since it was being used by multiple test files. Also I could not move that to a test file because there is a lint rule banning exports from test file :slightly_smiling_face:

Reply:
Files have been moved to relevant test locations after our discussion!